### DISCUSSION TITLE
Projects to add to the awesome-dioxus

### DISCUSSION THREAD

**Discussion by jkelleyrtp** [DioxusLabs/dioxus#184] [Show and tell]
List of projects pending being added to awesome-dioxus:

- [ ] https://github.com/houseabsolute/dioxus-heroicons
- [ ] https://github.com/rustkid/recorder

**Comment by rMazeiks** [comment]
I don't know if this is sufficiently awesome, but here's my take on (S)CSS-in-RS:

https://crates.io/crates/sir
